### PR TITLE
Adding a global RDS example

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: aws-ts-global-rds-cluster/index.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+});
+
+const secondaryCluster = new aws.rds.Cluster(`${secondaryRegion}-cluster`, {
+    globalClusterIdentifier: globalCluster.globalClusterIdentifier,
+    engineMode: "global",
+    engine: "aurora",
+    skipFinalSnapshot: true,
+}, {
+    provider: provider,
+    dependsOn: [provider, cluster],
```
Comment (lukehoban) [inline aws-ts-global-rds-cluster/index.ts]: Does this really need to depend on either of these?

File: aws-ts-global-rds-cluster/index.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+});
+
+const secondaryClusterInstance = new aws.rds.ClusterInstance(`${secondaryRegion}-cluster-instance`, {
+    engine: "aurora",
+    instanceClass: aws.rds.InstanceTypes.R5_Large,
+    clusterIdentifier: secondaryCluster.clusterIdentifier,
+}, {
+    provider: provider,
+    dependsOn: [secondaryCluster],
```
Comment (lukehoban) [inline aws-ts-global-rds-cluster/index.ts]: This dependency is implicit - so not needed here.

File: aws-ts-global-rds-cluster/index.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+const clusterInstance = new aws.rds.ClusterInstance("primary-cluster-instance", {
+    engine: "aurora",
+    instanceClass: aws.rds.InstanceTypes.R5_Large,
+    clusterIdentifier: cluster.clusterIdentifier,
+});
+
+let clusterEndpoints = new Map()
+
+// Get the primary cluster endpoint - this is used for writes
+// export const primary = clusterInstance.endpoint;
```
Comment (lukehoban) [inline aws-ts-global-rds-cluster/index.ts]: I assume this comment is no longer needed?

File: aws-ts-global-rds-cluster/index.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    skipFinalSnapshot: true,
+});
+
+const clusterInstance = new aws.rds.ClusterInstance("primary-cluster-instance", {
+    engine: "aurora",
+    instanceClass: aws.rds.InstanceTypes.R5_Large,
+    clusterIdentifier: cluster.clusterIdentifier,
+});
+
+let clusterEndpoints = new Map()
```
Comment (mikhailshilkov) [inline aws-ts-global-rds-cluster/index.ts]:
```suggestion
const clusterEndpoints = new Map();
```

File: aws-ts-global-rds-cluster/index.ts [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+}, {
+    provider: provider,
+    dependsOn: [secondaryCluster],
+});
+
+// Get the secondary cluster endpoint
+// export const secondary =  secondaryClusterInstance.endpoint;
+clusterEndpoints.set(`secondary-${secondaryRegion}`, secondaryClusterInstance.endpoint);
+
+for (let entry of clusterEndpoints.entries()) {
```
Comment (mikhailshilkov) [inline aws-ts-global-rds-cluster/index.ts]:
```suggestion
for (const entry of clusterEndpoints.entries()) {
```

File: aws-ts-global-rds-cluster/index.ts [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    exports[entry[0]] = entry[1]
```
Comment (mikhailshilkov) [inline aws-ts-global-rds-cluster/index.ts]:
```suggestion
    exports[entry[0]] = entry[1];
```

File: aws-ts-global-rds-cluster/index.ts [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    engine: "aurora",
+    instanceClass: aws.rds.InstanceTypes.R5_Large,
+    clusterIdentifier: secondaryCluster.clusterIdentifier,
+}, {
+    provider: provider,
+    dependsOn: [secondaryCluster],
+});
+
+// Get the secondary cluster endpoint
+// export const secondary =  secondaryClusterInstance.endpoint;
```
Comment (mikhailshilkov) [inline aws-ts-global-rds-cluster/index.ts]: Why not just use this normal export syntax? I guess you started with a variable number of clusters, but now you have two.

File: aws-ts-global-rds-cluster/index.ts [hunk 8]
```diff
@@ -0,0 +1,66 @@
+import {Config, interpolate} from "@pulumi/pulumi";
```
Comment (mikhailshilkov) [inline aws-ts-global-rds-cluster/index.ts]: Please fix all [linting errors](https://travis-ci.com/pulumi/examples/builds/130818198#L1039-L1046)

**Pull request closed without merging**